buildSrc/src/main/kotlin/Project.kt

```diff
@@ -12,6 +12,11 @@ object AndroidVersions {
     const val targetSdkVersion = 30
     const val compileSdkVersion = 31
   }
+  object ExampleApp {
+    const val minSdkVersion = 23
+    const val targetSdkVersion = 31
+    const val compileSdkVersion = 31
```

Comment:
I guess nice to tests with the latest version.
```suggestion
    const val targetSdkVersion = 33
    const val compileSdkVersion = 33
```